phoenicis-repository/src/main/java/org/phoenicis/repository/RepositoryConfiguration.java

```diff
@@ -84,11 +81,22 @@ public RepositoryManager repositoryManager() {
                 backgroundRepositoryFactory());
 
         // set initial repositories
-        repositoryManager.addRepositories(this.loadRepositoryLocations().toArray(new RepositoryLocation[0]));
+        repositoryManager.addRepositories(this.repositoryLocationLoader().loadRepositoryLocations().toArray(new RepositoryLocation[0]));
```

Comment:
Can we alternatively pass the `RepositoryLocationLoader` to the `DefaultRepositoryManager` constructor?
I believe this may be a more elegant design that what we have currently (i.e. `repositoryManager.addRepositories`)

Reply:
The would be more elegant, but I'm not sure that it won't break anything

Reply:
And I also think that the interface of RepositoryManager needs to be changed